### DISCUSSION TITLE
how to load android arm?

### DISCUSSION THREAD

**Discussion by Jokky6** [Vector35/binaryninja-api#3201] [Q&A] [answered]
when I use binary ninja ,how to import JavaVM like ida?

**Comment by psifertex** [comment, marked as answer]
JVM is not currently supported. See https://binary.ninja/faq/ for the list of official architectures and https://github.com/Vector35/community-plugins/#binary-ninja-plugins for the list of third party plugins. 

More specifically, virtual architectures such as JVM OR dotNET are not architecturally suited toward how binary ninja does its analysis. It may become possible once the "function at a a time" lifting feature is complete. On mobile so I can't link to it now but it's tracked in the issue database here on GH.
